CMakeLists.txt

```diff
@@ -861,11 +864,26 @@ if (NOT EVENT__DISABLE_OPENSSL)
 
     include_directories(${OPENSSL_INCLUDE_DIR})
 
-    list(APPEND SRC_OPENSSL bufferevent_openssl.c)
+    list(APPEND SRC_OPENSSL bufferevent_openssl.c bufferevent_ssl.c)
     list(APPEND HDR_PUBLIC include/event2/bufferevent_ssl.h)
     list(APPEND LIB_APPS ${OPENSSL_LIBRARIES})
 endif()
 
+if (NOT EVENT__DISABLE_MBEDTLS)
+    find_package(MbedTLS REQUIRED)
```

Comment:
```
CMake Warning (dev) at /usr/local/share/cmake-3.17/Modules/FindPackageHandleStandardArgs.cmake:272 (message):
  The package name passed to `find_package_handle_standard_args` (mbedTLS)
  does not match the name of the calling package (MbedTLS).  This can lead to
-- Found mbedTLS: /usr/include (found version "2.8.0") 
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindMbedTLS.cmake:127 (find_package_handle_standard_args)
  CMakeLists.txt:873 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- mbed TLS include: /usr/include
-- mbed TLS lib: /usr/lib/x86_64-linux-gnu/libmbedtls.so;/usr/lib/x86_64-linux-gnu/libmbedcrypto.so;/usr/lib/x86_64-linux-gnu/libmbedx509.so
```

will be addressed on merge

Reply:
fixed by modifying package name in `FindMbedTLS.cmake`